mixer/template/tracespan/tracespan.proto

```diff
@@ -115,4 +115,52 @@ message Template {
     // not provided for server spans.
     // Optional
     bool client_span = 9;
```

Comment:
See this is not part of your PR, so already defined as bool - but still think it should be changed as spans are not just client or server.

Reply:
Enum is not supported yet in Mixer expression language, and it is not easy to add. Bool is just temporary work around. Once enum is supported, we can deprecate this field.